podaac/subsetter/subset.py

```diff
@@ -322,8 +323,12 @@ def get_coord_variable_names(dataset):
         Tuple of strings, where the first element is the lat coordinate
         name and the second element is the lon coordinate name
     """
-    possible_lat_coord_names = ['lat', 'latitude', 'y']
-    possible_lon_coord_names = ['lon', 'longitude', 'x']
+
+    possible_lat_coord_names = ['lat', 'latitude', 'y'] + dataset.cf.coordinates.get('latitude', []) + dataset.cf.axes.get('Y', [])
```

Comment:
Is it possible to use xarray-cf entirely for determining coordinate variables? Or is the hardcoded list `['lat', 'latitude', 'y']` still needed in some cases?

Reply:
> Is it possible to use xarray-cf entirely for determining coordinate variables? Or is the hardcoded list `['lat', 'latitude', 'y']` still needed in some cases?

i didn't try without it i can test to see if things pass without it

Reply:
There's also a way to set custom criteria
https://cf-xarray.readthedocs.io/en/latest/custom-criteria.html

The way I'm reading things, we should always refer to `dataset.cf.coordinates['latitude']`. If we have a dataset where the "standard" criteria doesn't find the coord variable, we should add a rule using the custom criteria. 

And I'd also go as far as calling https://cf-xarray.readthedocs.io/en/latest/generated/xarray.Dataset.cf.add_canonical_attributes.html#xarray.Dataset.cf.add_canonical_attributes on the dataset to add CF attributes to the result